lib/nostrum/util.ex

```diff
@@ -4,29 +4,10 @@ defmodule Nostrum.Util do
   """
 
   alias Nostrum.{Api, Constants}
-  alias Nostrum.Shard.Stage.Producer
   alias Nostrum.Struct.Snowflake
 
   require Logger
 
-  @doc """
```

Comment:
I'm okay with removing this, but if we're going to we should add a note somewhere so those creating their own custom pipelines know what they're supposed to subscribe to.

Reply:
I'm not sure on the procedure for adding a customer consumer. It looks like the directions in the README docs are outdated. If you can list out the procedure, I can quickly follow up with a fix. Might be a documentation change or reverting this commit depending on what is said.

Reply:
The [documentation](https://kraigie.github.io/nostrum/consumers.html#implementing-custom-consumers) looks up to date with me, besides there being an error (`To do so, simply add custom_consumer: false to your config.` -> `costum_consumer: true`). The bigger issue is that the implementation no longer works as a lot of the processes and supervision trees have changed since it was implemented. Also in the original implementation we weren't super thorough in making sure we didn't start processes that we didn't need.

I'd say for now lets leave this removed, and when the feature is re-implemented we can worry about it then.